packages/kbn-reporting/server/config_schema.ts

```diff
@@ -66,10 +66,10 @@ const CsvSchema = schema.object({
   useByteOrderMarkEncoding: schema.boolean({ defaultValue: false }),
   scroll: schema.object({
     duration: schema.string({
-      defaultValue: '30s', // this value is passed directly to ES, so string only format is preferred
+      defaultValue: 'auto', // other possible values are passed directly to ES, so string only format is preferred
```

Comment:
I think we shouldn't change the default in this PR. For now, we should support it and document it, and users should opt-in to this setting. It can become the new default after it proves itself and there is popular demand.